.prettierrc

```diff
@@ -7,11 +7,12 @@
 		{
 			"files": ["*.svelte"],
 			"options": {
-				"svelteBracketNewLine": true
+				"bracketSameLine": false
```

Comment:
Strange, both should work. The former is deprecated in favor of the latter, so this is a good change anyway.